code/modules/mob/living/carbon/alien/alien_base.dm

```diff
@@ -220,14 +220,18 @@ Des: Removes all infected images from the alien.
 	return
 
 /mob/living/carbon/alien/canBeHandcuffed()
-	return 1
+	return TRUE
 
-/mob/living/carbon/alien/proc/updatePlasmaDisplay()
-	if(hud_used) //clientless aliens
-		hud_used.alien_plasma_display.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'> <font face='Small Fonts' color='magenta'>[getPlasma()]</font></div>"
-		hud_used.alien_plasma_display.maptext_x = -3
+/* Although this is on the carbon level, we only want this proc'ing for aliens that do have this hud. Only humanoid aliens do at the moment, so we have a check
+and carry the owner just to make sure*/
+/mob/living/carbon/proc/updateplasmadisplay(mob/owner)
+	RegisterSignal(owner, COMSIG_MIND_TRANSER_TO)
+	if(!hud_used || !isalien(owner)) //clientless aliens or non aliens
+		return
+	hud_used.alien_plasma_display.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'> <font face='Small Fonts' color='magenta'>[get_plasma()]</font></div>"
+	hud_used.alien_plasma_display.maptext_x = -3
 
-/mob/living/carbon/alien/larva/updatePlasmaDisplay()
+/mob/living/carbon/alien/larva/updateplasmadisplay()
```

Comment:
Please change this to snake_case if you're changing the casing

Reply:
done